### PR TITLE
Add file_limit support for RedHat platforms

### PR DESCRIPTION
**Pull request by fcharlier**

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -106,14 +106,47 @@
...
+        } ->
+        file { '/etc/systemd/system/rabbitmq-server.service.d/limits.conf':
+          content => template('rabbitmq/rabbitmq-server.service.d/limits.conf'),
+          owner   => '0',
+          group   => '0',
+          mode    => '0644',
+          notify  => Exec['/usr/bin/systemctl daemon-reload'],
+        }
+        exec { '/usr/bin/systemctl daemon-reload':
+          notify      => Class['Rabbitmq::Service'],
```
Comment (DavidS) [inline manifests/config.pp]: Please change this to use a rabbitmq specific title, so it avoids a potential collision with other (as yet unknown) modules like so:

    exec { 'rabbitmq-systemd-reload': command => '/usr/bin/systemctl daemon-reload', #...


and change all dependencies to use `Exec['rabbitmq-systemd-reload']`

**Comment by DavidS** [comment]
After fixing the exec issue, this can be merged.

**Comment by fcharlier** [comment]
@DavidS thanks for the review, fixed the exec title and rebased !

**Comment by DavidS** [comment]
Excellent, thanks!

**Pull request merged**